samples/nrf9160/modem_shell/src/link/link_api.h

```diff
@@ -13,6 +13,9 @@
 
 #include "mosh_defines.h"
 
+/** SNR offset value that is used when mapping to dBs  */
+#define LINK_SNR_OFFSET_VALUE 25
```

Comment:
Just wondering why this is 25 and not 24? The AT command documentation says:

_The index value of SNR can be converted to decibel with the following formula:
Index – 24 = dB. For example, 1 – 24 = –23 dB and 47 – 24 = 23 dB._

Reply:
A good point. Don't remember how did we originally came to value of 25.
That value wasn't set in this PR. I will change this now.

Reply:
@tokangas. Where is that snr formula said? Actually, I think that the value 25 was based on at-cmd excel.

There is for example:
for value "0: snr < -24 "
i.e. 0-25= -25 is less than (< -24) => in limits. When subtracting with 24 it is equal to -24 that is not less than 24 (<24).






































































Actually, not sure if 25 is incorrect, might be so that because

Reply:
It's in the AT command reference manual, for example here (see chapter 8.12): https://infocenter.nordicsemi.com/pdf/nrf91_at_commands_v1.9.pdf

Reply:
OK, I will change to offset of 24. Still not get their examples there e.g. for:
1 – When −24 dB ≤ SNR < −23 dB

=> 1 - 24 = -23db which is out of boundaries of SNR.